content/articles/index.md

```diff
@@ -0,0 +1,316 @@
+            }
+        }
+        catch (IOException e)//catches exception in the try block
+        {
+            e.printStackTrace();//Prints this throwable and its backtrace
+        }
+    }
+}
+```
+
+The output of the code is as shown below.
+
+```bash
+Name: Student Name,Faculty: Faculty, Registration: Registration No, Fees Balance: Fees Balance, Campus:  Campus
+Name: Stanley Kuria,Faculty: Education, Registration: 37916808, Fees Balance: 3150, Campus:  Main
+Name: Davidson Kanyua,Faculty: Computing, Registration: 38094044, Fees Balance: 8500, Campus:  Meru
+Name: Tess Wambui,Faculty: Engineering, Registration: 21666504, Fees Balance: 13500, Campus:  Nairobi
+Name: Esther Njoroge,Faculty: Health, Registration: 37809504, Fees Balance: 5500, Campus:  Nakuru
+```
+
+### Java scanner class.
+It is a class in the `Java.util` bundle, used to get input from the user. Breaks information into tokens, utilizing the delimiter pattern. It is the most straightforward method to read input in java code.
+We used the scanner class to read CSV files in Java in the code below.
+
+```Java
+import java.util.Scanner;//contains scanner class
+import java.io.*;
+class mycsvread
+{
+    public static void main(String[] args) throws FileNotFoundException //This exception will be thrown when a file with the specified pathname does not exist
+    {
+        Scanner myscanner = new Scanner(new File("C:\\Users\\davis\\eclipse-workspace\\CSVdocuments\\commaseperated.csv"));//Creates a new File
+        myscanner.useDelimiter(", ");   //Sets the delimiter
+        while (myscanner.hasNext())  //Returns a boolean(true) if this scanner has another token in its input
+        {
+            System.out.println(myscanner.next());//next complete scanner is returned here
+        }
+        myscanner.close();  //myscanner is closed
+    }
+}
+```
+
+The output is as shown below.
+
+```bash
+Student Name,Faculty,Registration No,Fees Balance,Campus
+Stanley Kuria,Education,37916808,3150,Main
+Davidson Kanyua,Computing,38094044,8500,Meru
+Tess Wambui,Engineering,21666504,13500,Nairobi
+Esther Njoroge,Health,37809504,5500,Nakuru
+```
+### Using openCSV API.
+OpenCSV is an outsider API utilized to peruse different adaptations of the comma-separated document. It supports all basic Comma-separated operations.
+
+OpenCSV is used since Java does not provide native support to read Comma-Separated values.
+
+#### Important OpenCSV classes
+- `CSVWrite` -  enables writing data into a CSV file.
+- `CSVToBean` - utilized to populate Java Application with CSV data.
+- `BeanToCSV` - applied to export information from JavaBean to Comma-seperated value file.
+Comma-separated values are read in two ways, namely:
+- By line: In this case, the comma-separated values are consecutively read from one line to another.
+- All data: utilizes readAll() technique to peruse all records at once.

```

Comment:
Are these two also openCSV classes?

Reply:
@paulodhiambo . The two  are  openCSV classes. Those  ways of reading a CSV file in java.